feconf.py

```diff
@@ -83,6 +83,10 @@
     'TextInput': {
```

Comment:
Should we drop this one?

Reply:
OK. So we still have old functionality in place which uses this values (LDAStringClassifier and all classifier methods). I think it would be a good idea to remove this value along with old infrastructure.